paystack/build.gradle

```diff
@@ -31,6 +37,12 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$versions.kotlin"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$versions.coroutines"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$versions.coroutines"
+
+    testImplementation 'org.robolectric:robolectric:4.4'
+    testImplementation "org.mockito:mockito-core:3.8.0"
+    testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
+    testImplementation "androidx.test.ext:junit-ktx:1.1.2"
```

Comment:
Hi Michael - it might be a good idea to follow the pattern being used above? 

```testImplementation 'org.robolectric:robolectric:$versions.robolectric'```